src/components/PathwayProvider.tsx

```diff
@@ -29,7 +29,8 @@ interface PathwayProviderProps {
 
 export const PathwayProvider: FC<PathwayProviderProps> = memo(({ children }) => {
   const [pathways, setPathways] = useState<Pathway[]>([]);
-  const service = useGetPathwaysService(config.get('demoPathwaysService'));
+  const pathwayObject = {} as Pathway;
```

Comment:
The approach of creating a dummy object to pass into the generic service feels a little strange, but I'm ok with it if necessary. I do think because TypeScript is bolted on instead of being a core part of the language, we can't do things as cleanly as we might like. That said, if it's possible to either 
A) Add generic typing to the useGetService function signature, so it would look like `useGetService<Pathway>(...)`
or
B) pass in the type `Pathway` directly instead of an instance of it
Either of those would be ideal IMO.

Reply:
Yeah, totally agree. I think I can figure out option (A).